Resources/ServerInfo/Rules.txt

```diff
@@ -79,7 +79,7 @@ If you regularly play Security or Command roles and got this far, we applaud you
     - Do not willingly and openly cooperate with terrorists/antagonists. Do not give away your objective items. Some leeway is given to making deals with antagonists if the deal benefits the safety or situation of the station as a whole and not just yourself.
     - Uphold the Law & maintain order. Do not engage in lawbreaking activity or troublemaker behavior. Security is expected to intervene into criminal activity where possible. Heads of Staff are at minimum expected to report criminal activity to Security.
     - Do not immediately abandon your position as a Command role and go do whatever you want instead of managing your department/the station. Do not abuse your position or use it to make arbitrary choices to the detriment of the station.
-    - Do not hire random crew to be your bodyguards or promote random to Captain or a Head of Staff at random. If you need bodyguards, talk to your security department. If you need a new Command role, talk to the personnel in that related department.
+    - Do not hire random crew to be your bodyguards or promote random crew to Captain or a Head of Staff at random. If you need bodyguards, talk to your security department. If you need a new Command role, talk to the personnel in that related department.
```

Comment:
```suggestion
    - Do not hire random crew to be your bodyguards or promote to Captain or a Head of Staff at random. If you need bodyguards, talk to your security department. If you need a new Command role, talk to the personnel in that related department.
```

Reply:
wouldn't it be better to just put "promote them to"

Reply:
> wouldn't it be better to just put "promote them to"

I agree with that, yes.